pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -2234,6 +2321,12 @@
     ::  save duct for future %give to unix
     ::
     =.  outgoing-duct.server-state.ax  duct
+    ::  send all cache bindings to runtime
+    ::
+    =/  cache-moves=(list move)
+      %+  turn  ~(tap by cache-bindings.server-state.ax)
+      |=  [url=@t =path]
+      [duct %give %cache-changed url path]
```

Comment:
What is the expected behavior of the http i/o driver as relates to cache bindings/entries?

Reply:
For now, the behavior is to discard the gift. Can we discuss the intended future behavior of how the HTTP driver should utilize the map that is published to it sometime soon? There are a few viable implementation paths that could make use of that data efficiently, but it will be in a separate grant.